src/uu/w/src/w.rs

```diff
@@ -283,6 +336,15 @@ mod tests {
         )
     }
 
+    #[test]
+    fn test_format_time_elapsed() {
+        let td = Duration::new(60 * 60 * 18 + 60 * 18, 0);
+        assert_eq!(
+            format_time_elapsed(td, false).unwrap(),
+            String::from("18:18m")
+        );
```

Comment:
What's the expected output if you pass `true` to the `format_time_elapsed()` function?